### PR TITLE
Open separate tab for cloud analytics link

### PR DESCRIPTION
**Pull request by gildub**
Resolves https://github.com/konveyor/virt-ui/issues/226

### CODE REVIEW

**Comment by gildub** [comment]
@mturley, for the navigation part it turns out that NavItem doesn't has a way to pass target attribute.
So one solution is to use <a href> and tweak css props to make it look like a NavItem.
What do you think?

**Comment by mturley** [comment]
@gildub I think we can just remove the `to` prop from that `NavItem` and render a normal `<a href="..." target="...">` as a child of it. Our other NavItems render `<NavLink>` children (from react-router-dom) which end up rendering `<a>` elements as well. I think the `to` prop is just a shorthand for this.

**Comment by gildub** [comment]
@mturley, Using an anchor element within NavItem doesn't play very well. I used onClick instead.

**Comment by mturley** [comment]
Ok, that works.

File: src/app/Providers/components/CloudAnalyticsInfoAlert.tsx [hunk 1]
```diff
@@ -9,7 +9,11 @@ const CloudAnalyticsInfoAlert: React.FunctionComponent = () => {
     LocalStorageKey.isProvidersPageMAAlertHidden
   );
 
-  const link = <a href={CLOUD_MA_LINK.href}>{CLOUD_MA_LINK.label}</a>;
+  const link = (
+    <a href={CLOUD_MA_LINK.href} target="_blank" rel="noreferrer">
```
Comment (mturley) [inline src/app/Providers/components/CloudAnalyticsInfoAlert.tsx:13]: Good call on `rel="noreferrer"`, I always forget that.

**Review by mturley — APPROVED** [review]

**Pull request merged**